composer.json

```diff
@@ -29,7 +29,6 @@
         "guzzlehttp/guzzle": "^6.3",
         "knplabs/console-service-provider": "^2.1",
         "league/commonmark": "^0.17.0",
-        "psr/container": "^1.0",
```

Comment:
Unrelated to this PR, but it _should_ be included since `AppKernel` implements `ContainerInterface`.